server/tests/db/test_01_es6.js

```diff
@@ -307,10 +307,21 @@ tests.NumberIsNaN = function() {
   console.assert(!Number.isNaN(' '), 'Number.isNaN " "');
 };
 
+tests.NumberIsInteger = function() {
+  console.assert(Number.isInteger(-17), 'Number.isInteger -17');
+  console.assert(Number.isInteger(Math.pow(2, 64)), 'Number.isInteger 2**64');
+  console.assert(-Number.isInteger(Math.pow(2, 64)), 'Number.isInteger -2**64');
+  console.assert(!Number.isInteger(NaN), 'Number.isInteger NaN');
+  console.assert(!Number.isInteger(Infinity), 'Number.isInteger Infinity');
+  console.assert(!Number.isInteger('3'), 'Number.isInteger "3"');
+  console.assert(!Number.isInteger(3.1), 'Number.isInteger 3.1');
+  console.assert(Number.isInteger(3.0), 'Number.isInteger 3.0');
+};
+
 tests.NumberIsSafeInteger = function() {
   console.assert(Number.isSafeInteger(3), 'Number.isSafeInteger 3');
-  console.assert(!Number.isSafeInteger(Math.pow(2, 53)), 'Number.isSafeInteger 2^53');
-  console.assert(Number.isSafeInteger(Math.pow(2, 53) - 1), 'Number.isSafeInteger 2^53-1');
+  console.assert(!Number.isSafeInteger(Math.pow(2, 53)), 'Number.isSafeInteger 2**53');
+  console.assert(Number.isSafeInteger(Math.pow(2, 53) - 1), 'Number.isSafeInteger 2**53-1');
```

Comment:
Line length (two lines here, and ~14 more below).

Reply:
The Writely team (and others at Google) agreed that line length didn't apply to unit tests.  In general I think that makes things more readable.  As a specific example, imagine what this file would look like with 80 columns:
https://github.com/google/diff-match-patch/blob/master/javascript/tests/diff_match_patch_test.js#L406

Every year or so some open source contributor attempts to line wrap that file (or its cousins in other languages) and the result is nasty.

Reply:
Well, tastes in aesthetics vary, but in this case I'd probably indent some of those tests like this:
```JS
  assertEquivalent(
      [[DIFF_DELETE, 'Apple'],
       [DIFF_INSERT, 'Banana'],
       [DIFF_EQUAL, 's are a'],
       [DIFF_INSERT, 'lso'],
       [DIFF_EQUAL, ' fruit.']],
      dmp.diff_main('Apples are a fruit.',
                    'Bananas are also fruit.', false));
```

(Though I still think it looks weird putting the expected output first.  Cursed yoda conditionals.)

In any case, `test_01_es6.js` is otherwise (mostly) in compliance with the JS style guide,  which enumerates the allowable exceptions to the 80-column limit—none of which appear to apply here.